LanguageFeatures/Records/canonicalization_A01_t02.dart

```diff
@@ -33,28 +33,47 @@ import "../../Utils/expect.dart";
 
 main() {
   if (!isJS) {
-    var r1 = ("a", 1, const [42], const {"x": 0}, const {1, 2, 3});
+    var r1 = const ("a", 1, [42], {"x": 0}, {1, 2, 3});
     var r2 = const ("a", 1.0, [42], {"x": 0}, {1, 2, 3},);
+    var r3 = ("a", 1.0, const [42], const {"x": 0}, const {1, 2, 3},);
     Expect.isFalse(identical(r1, r2));
+    Expect.isFalse(identical(r1, r3));
   }
 
-  var r3 = (s: "a", n: 1, l: const [42], m: const {"x": 0}, set: const {1, 2, 3});
-  var r4 = const (notS: "a", n: 1, l: [42], m: {"x": 0}, set: {1, 2, 3});
-  Expect.isFalse(identical(r3, r4));
+  var r4 = const (s: "a", n: 1, l: [42], m: {"x": 0}, set: {1, 2, 3});
+  var r5 = const (notS: "a", n: 1, l: [42], m: {"x": 0}, set: {1, 2, 3});
+  var r6 =
+    (notS: "a", n: 1, l: const [42], m: const {"x": 0}, set: const {1, 2, 3});
+  Expect.isFalse(identical(r4, r5));
+  Expect.isFalse(identical(r4, r6));
 
-  var r5 = ("a", 1, l: const [42], m: const {"x": 0}, set: const {1, 2, 3});
-  var r6 = const (1, l: [42], m: {"x": 0}, set: {1, 2, 3});
-  Expect.isFalse(identical(r5, r6));
-
-  var r7 = ("a", 1, l: const [42], m: const {"x": 0}, set: const {1, 2, 3});
-  var r8 = const ("a", 1, l: [42], m: {"x": 0}, set: {1, 2, 3, 4});
+  var r7 = const ("a", 1, l: [42], m: {"x": 0}, set: {1, 2, 3});
+  var r8 = const (1, l: [42], m: {"x": 0}, set: {1, 2, 3});
+  var r9 = const (1, l: const [42], m: const {"x": 0}, set: const {1, 2, 3});
```

Comment:
Ah, of course! One more way to do `const`. No need to do that everywhere, but it's nice that we have one of them.